tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "noUnusedLocals": true,
     "noImplicitAny": false,
     "target": "es5",
-    "types": ["jest"]
+    "types": ["jest", "node"]
```

Comment:
Does the script not work without compiling with the node types?

Reply:
Got it, so it does not unfortunately. It complained about the type of `process` 🤔

Reply:
More specifically:
```
create_new_component.ts (3,29): Cannot find name 'process'. Do you need to install type definitions for node? Try `npm i @types/node` and then add `node` to the types field in your tsconfig. (2591)
```

Reply:
Ah I didn't realize the script relied on that.

Reply:
We spoke about this in person and @cozmo suggested adding a separate tsconfig for these scripts. That seems like a good approach to me.

Reply:
Thanks to our discussion offline, fixed in 3c101ec

Reply:
👌